.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -44,6 +44,7 @@ env:
   CUSTOM_TEST_ARTIFACT_BUILD_DIR: build/custom_test_artifacts
   ALPINE_IMAGE: "308535385114.dkr.ecr.us-east-1.amazonaws.com/tool/alpine"
   PR_LABELS: ${{ toJson(github.event.pull_request.labels.*.name) }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I only changed the linux template, I'm not sure the windows tests are running the torchhub tests. @seemethere would you know how I could confirm that?

Reply:
We'll also need to add `-e GITHUB_TOKEN` to the docker runs as well so that it gets propagated to the container as well

Reply:
Actually nvm, we already do this since we create an env file that gets auto-included in our docker runs:

https://github.com/pytorch/pytorch/blob/67bd2a31b52b01b83a72bb0c8d1c2ef4d95d103b/.github/templates/common.yml.j2#L84-L86